modules/kafka-cluster/inputs.tf

```diff
@@ -30,5 +23,26 @@ variable "region" {
   description = "The cloud service provider region where the Kafka cluster is running - see supported list - https://docs.confluent.io/cloud/current/clusters/regions.html#cloud-providers-and-regions"
   type        = string
   default     = "asia-southeast2"
+}
+
+variable "cluster_type" {
+  description = "The value can be either basic, standard, and dedicated."
+  type        = string
+  default     = "basic"
+  validation {
+    condition     = can(regex("^(basic|standard|dedicated)$", var.cluster_type))
+    error_message = "Invalid input, only mentioned options can be used: \"basic\", \"standard\", \"dedicated\""
+  }
+}
 
+variable "dedicated_cluster_cku" {
+  description = "The number of Confluent Kafka Units (CKUs) for Dedicated cluster types. The minimum number of CKUs for SINGLE_ZONE dedicated clusters is 1 whereas MULTI_ZONE dedicated clusters must have more than 2 CKUs."
```

Comment:
"more than 2 CKUs" (which means >= 3) or "at least 2 CKUs" (which means >= 2)?

Reply:
which means >= 2 . Have updated the description too